### ISSUE TITLE
document.write spof

### ISSUE THREAD

**Issue by ddunlop** [senthilp/spofcheck#1]
The rules don't catch single points of failure that make use document.write, such as the old google analytics tags.

``` html
<script type="text/javascript">
  var gaJsHost = (("https:" == document.location.protocol) ?
  "https://ssl." : "http://www.");
   document.write(unescape("%3Cscript src='" + gaJsHost +
  "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
```

**Comment by senthilp** [comment]
This may not be a problem anymore with browsers coming up with Interventions - [Intervening against document.write()](https://developers.google.com/web/updates/2016/08/removing-document-write).
